build-tools/automation/yaml-templates/install-dotnet-tool.yaml

```diff
@@ -23,5 +23,3 @@ steps:
       --version ${{ parameters.version }}
       --add-source "https://api.nuget.org/v3/index.json"
 
-- script: echo "##vso[task.prependpath]$(Agent.ToolsDirectory)"
-  displayName: add $(Agent.ToolsDirectory) to path
```

Comment:
This solves the problem with `%PATH%`:

![image](https://github.com/user-attachments/assets/171d6027-e969-4774-85f0-34fb95a4bf70)

Going to rebase, we have this change in main now.